src/main/java/org/folio/event/handler/HoldingUpdateAsyncRecordHandler.java

```diff
@@ -141,7 +141,8 @@ private Pair<Boolean, List<PoLine>> updatePoLinesInstance(HoldingEventHolder hol
           .withTitleOrPackage(InventoryUtils.getInstanceTitle(holder.getInstance()))
           .withPublisher(InventoryUtils.getPublisher(holder.getInstance()))
           .withPublicationDate(InventoryUtils.getPublicationDate(holder.getInstance()))
-          .withContributors(InventoryUtils.getContributors(holder.getInstance()));
+          .withContributors(InventoryUtils.getContributors(holder.getInstance()))
+          .getDetails().withProductIds(InventoryUtils.getProductIds(holder.getInstance()));
         updatedPoLines.add(poLine);
```

Comment:
Validated & normalized version will look something like in `mod-orders`: 

- `OrderLinePatchOperationService.updatePoLineWithInstanceRecordInfo(JsonObject lookupObj, PoLine poLine, RequestContext requestContext)`

![image](https://github.com/user-attachments/assets/1fe0702d-eec0-47e8-8026-3cb1c72be5ce)